Compliation Error on OSX - QUIRC

Receive the following error on Mojave even though WITH_SCANNER should be set to OFF by default:

[ 57%] Building C object external/CMakeFiles/quirc.dir/quirc/lib/quirc.c.o
Scanning dependencies of target openpgp
/Users/hiro1/monero-gui/external/quirc/lib/quirc.c:126:22: error: result [ 57%] Generating monero-core.qm
of comparison of constant 8 with expression of type 'quirc_decode_error_t' is always true [-Werror,-Wtautological-constant-out-of-range-compare]
        if (err >= 0 && err < sizeof(error_table) / sizeof(error_table[0]))
                        ~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,6 +58,7 @@ if(NOT MANUAL_SUBMODULES)
 endif()
 
 add_subdirectory(monero)
+add_subdirectory(external)
 
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
@@ -560,6 +561,5 @@ if (HIDAPI_FOUND OR LibUSB_COMPILE_TEST_PASSED)
   endif()
 endif()
 
-add_subdirectory(external)
 add_subdirectory(translations)
 add_subdirectory(src)

